### PR TITLE
chore: allow manual e2e workflow runs

### PR DESCRIPTION
**Pull request by sebastianquek**
GitHub actions now support manually triggered workflows. Have enabled this for e2e tests so that we can run it on the fly.
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch

Example:
image

### CODE REVIEW

File: .github/workflows/test_e2e.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@ on:
       - master
     types:
       - completed
+  # Allow manual runs
+  workflow_dispatch:
```
Comment (ilhaoni) [inline .github/workflows/test_e2e.yml:16]: hmm, i think may need the inputs field here, if we're pasting a url in? :O 

or currently its more to trigger it when needed~ xD

Comment (sebastianquek) [reply .github/workflows/test_e2e.yml:16]: trigger when needed, so no url needed, check out the code here: https://github.com/sebastianquek/github-actions-test/blob/master/.github/workflows/test_env_var.yml

```
on:
  push:
  pull_request:
  workflow_dispatch:
```

Comment (ilhaoni) [reply .github/workflows/test_e2e.yml:16]: okk! got it haha~ thanks~

**Review by ilhaoni — APPROVED** [review]
LGTM! :D

**Pull request merged**